Dockerfile-generator

```diff
@@ -13,13 +13,19 @@ WORKDIR /code
 RUN apt-get install -y git imagemagick wget \
   && apt-get clean
 
-RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - \
+RUN curl -sL https://deb.nodesource.com/setup_16.x | bash - \
   && apt-get install -y nodejs \
   && apt-get clean
 
-RUN npm install -g npm@6.3.0
+RUN npm install -g npm@7.21.1
 
-RUN gem install bundler --version '>= 2.1.4' \
+RUN  curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | gpg --dearmor | tee /usr/share/keyrings/yarnkey.gpg >/dev/null \
```

Comment:
Just a question, you may have a valid reason to do this.

But I was wondering since we already have `npm` at this point, why not just do?
```bash
npm install -g yarn
```

Reply:
That was what I initially have done, but then I saw that in the  output of the docker build command (after adding the NodeJS repository) there's this:

 ```bash
## Run `sudo apt-get install -y nodejs` to install Node.js 16.x and npm
## You may also need development tools to build native addons:
     sudo apt-get install gcc g++ make
## To install the Yarn package manager, run:
     curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | gpg --dearmor | sudo tee /usr/share/keyrings/yarnkey.gpg >/dev/null
     echo "deb [signed-by=/usr/share/keyrings/yarnkey.gpg] https://dl.yarnpkg.com/debian stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
     sudo apt-get update && sudo apt-get install yarn
 
```

But now I see that in fact in the [URL with the error from rails/decidim generator](https://yarnpkg.com/lang/en/docs/install/) says that the recommended way is through npm, and as that's also shorter and cleaner, I've changed it to that.